api/src/pcapi/routes/serialization/collective_bookings_serialize.py

```diff
@@ -95,6 +96,7 @@ class CollectiveBookingResponseModel(BaseModel):
     booking_amount: float
     booking_status_history: list[BookingStatusHistoryResponseModel]
     booking_identifier: str
+    banking_stats: financial_models.BankInformationStatus | None
```

Comment:
(détail) "stats" me fait penser à "statistics". ;)

```suggestion
    banking_status: financial_models.BankInformationStatus | None
```